sphinxcontrib/datatemplates/directive.py

```diff
@@ -60,14 +62,28 @@ def run(self):
                 line=self.lineno)
             return [error]
 
+        key = self.options.get('key')
+        include_context = self.options.get('include_context')
+
         data = self._load_data(env, data_source)
 
-        context = {
+        context = {}
+
+        if include_context:
+            try:
+                context.update(app.builder.html_context)
+            except KeyError:
+                app.warn('`include_context` set, but builder has no `html_context`.')
```

Comment:
Always making this a warning will cause it to fail for latex builds when -W is used. I suppose that's desirable, though?